drive/google/google.go

```diff
@@ -1,42 +1,145 @@
 package google
 
-import "github.com/asjoyner/shade/drive"
+import (
+	"bytes"
+	"context"
```

Comment:
Ah, neat. Done.

BTW, golang.org recommends 1.7.1 for new downloads. Is that the same as "tip?" Should we add a 1.7 Travis CI?